### PR TITLE
Fix a dependency issue for gitpython

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
(CI didn't run, so please don't merge before that happened)

**Comment by Odilhao** [comment]
[test rpm]

**Comment by Odilhao** [comment]
[test rpm]

File: packages/python-gitpython/python-gitpython.spec [hunk 1]
```diff
@@ -66,5 +67,8 @@ set -ex
 
 
 %changelog
+* Thu Mar 17 2022 Odilon Sousa <osousa@redhat.com> - 3.1.26-2
+- Release python-gitpython 3.1.26
```
Comment (ekohl) [inline packages/python-gitpython/python-gitpython.spec]: This doesn't sound like an accurate description

Comment (Odilhao) [reply packages/python-gitpython/python-gitpython.spec]: Yeah, fixing this one, my bad

**Review by ekohl — APPROVED** [review]

**Pull request merged**